{
    "error": "HTTPConnectionPool(host='118.89.230.52', port=80): Max retries exceeded with url: /about/details.docx (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7effc5bb6a90>, 'Connection to 118.89.230.52 timed out. (connect timeout=10)'))"
}